gallery/src/components/NxVulnerabilityIndicator/NxVulnerabilityIndicatorPage.tsx

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTable, NxCode, NxP, NxTextLink, NxH3 } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxVulnerabilityIndicatorBySeverityExample from './NxVulnerabilityIndicatorBySeverityExample';
+import NxVulnerabilityIndicatorByScoreExample from './NxVulnerabilityIndicatorByScoreExample';
+import NxVulnerabilityIndicatorListExample from './NxVulnerabilityIndicatorListExample';
+import NxVulnerabilityIndicatorTableExample from './NxVulnerabilityIndicatorTableExample';
+import NxVulnerabilityIndicatorWithTitleExample from './NxVulnerabilityIndicatorWithTitleExample';
+
+const nxVulnerabilityIndicatorBySeverityCode = require('./NxVulnerabilityIndicatorBySeverityExample?raw'),
+    nxVulnerabilityIndicatorByScoreCode = require('./NxVulnerabilityIndicatorByScoreExample?raw'),
+    nxVulnerabilityIndicatorListCode = require('./NxVulnerabilityIndicatorListExample?raw'),
+    nxVulnerabilityIndicatorTableCode = require('./NxVulnerabilityIndicatorTableExample?raw'),
+    nxVulnerabilityIndicatorWithTitleCode = require('./NxVulnerabilityIndicatorWithTitleExample?raw');
+
+const NxVulnerabilityIndicatorPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        <NxCode>NxVulnerabilityIndicator</NxCode> is an inline element used
+        to indicate via color the IQ policy vulnerability rating of the information to follow that follows it.
+      </NxP>
+      <NxP>
+        There are two scales to choose from: vulnerability rating by category, and vulnerability
+        rating by score. When using this component, it is expected that just one of the props will be passed. If both
```

Comment:
Category doesn't mean anything here, so this needs to be reworded.  How about something like

> The severity rating can either be specified directly, or a CVSS score may be specified from which the component will compute the rating.

Reply:
That seems right, thank you!

Reply:
> That seems right, thank you!

It doesn't appear that this change was actually made.